python/tests/res/enkf/test_enkf.py

```diff
@@ -231,4 +232,28 @@ def test_run_context(self):
             arg0 = run_context[0]
             arg2 = run_context.iensGet( 2 )
             #self.assertEqual( arg0 , arg2 )
+    
+    
+    def test_run_context_from_external_folder(self):
+        with TestAreaContext('enkf_test') as work_area:
+            work_area.copy_directory(self.case_directory_custom_kw)
+            res_config_path = work_area.getFullPath('snake_oil/snake_oil.ert')
+            
+            os.chdir('/tmp')
```

Comment:
Make a new `TestAreaContext` inside instead of hardcoding `/tmp` - that generally comes back to bite us with permission problems.